contracts/gov/user-keeper/GovUserKeeper.sol

```diff
@@ -505,6 +505,8 @@ contract GovUserKeeper is IGovUserKeeper, OwnableUpgradeable, ERC721HolderUpgrad
             return 0;
         }
 
+        ERC721Power nftContract = ERC721Power(nftAddress);
```

Comment:
`nftContract` can be created before the `if` above so save gas

Reply:
done